packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -104,4 +106,14 @@ export class ElectronActions {
       return obj.filePath || null
     })
   }
+
+  showSystemNotification (title: string, body: string) {
+    const notification = this.ctx.electronApi.createNotification(title, body, !this.isMac ? getPathToIcon('icon_64x64.png') : undefined)
```

Comment:
Why exclude Mac here?

Reply:
I added the icon because the Windows notifications weren't showing the Cypress shortcut logo in the notification. It looks weird when you do it on Mac because the logo appears twice in the notification (once where macOS puts it, and then once in the body of the notification). Upon further inspection though, the icon issue with Windows might be related to how our app is configured, so maybe I'll remove this completely until we figure that out. See https://github.com/cypress-io/cypress/issues/26786